app/helpers/ops_helper/role_rbac_details_helper.rb

```diff
@@ -0,0 +1,46 @@
+module OpsHelper::RoleRbacDetailsHelper
+  private
+
+  def role_rbac_details(role, rbac_menu_tree)
+    summary = []
+    summary.push(rbac_role_info_view(role, rbac_menu_tree))
+    summary.push(rbac_groups_using_role_info(role))
+    safe_join(summary)
```

Comment:
can we drop the array?
```suggestion
    safe_join([
      rbac_role_info_view(role, rbac_menu_tree),
      rbac_groups_using_role_info(role)
    ])
```